.github/workflows/selfservice-backend.yml

```
@@ -0,0 +1,91 @@
+name: Build - SelfService - Backend
+
+on:
+  push:
+    branches:
+      - "**"
+    paths:
+      - "Source/SelfService/Backend/**"
+      - ".github/workflows/selfservice-backend.yml"
+  pull_request:
+    types: [closed]
+
+env:
+  PRERELEASE_BRANCHES: "" # Comma separated list of prerelease branch names. 'alpha,rc, ...'
+  SOURCE_PATH: "Source/SelfService/Backend"
+  DOCKER_FILE: "Source/SelfService/Backend/Dockerfile"
+  DOCKER_IMAGE_TAG: "dolittle/studio/self-service-backend"
+  CONTAINER_REGISTRY: 508c17455f2a4b4cb7a52fbb1484346d.azurecr.io
```

Small tweaks, I did this yesterday (if you were curious)

```suggestion
DOCKER_IMAGE_TAG: "dolittle/self-service-backend"
CONTAINER_REGISTRY: docker.io
```

---

Does this mean that we're pushing this to docker hub instead?